### PR TITLE
Bugfixes after testing RA Locations

### PR DESCRIPTION
**Pull request by relaxnow**

### CODE REVIEW

**Comment by relaxnow** [comment]
@rjkip / @arothuis review plz?
plz?

File: src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Projector/RaLocationProjector.php [hunk 1]
```diff
@@ -37,13 +45,71 @@ public function __construct(RaLocationRepository $repository)
...
+        $raLocation = $this->repository->findByRaLocationId($raLocationId);
+
+        if (is_null($raLocation)) {
+            throw new RuntimeException(
+                'Tried to update an RA Locations contact information, but location could not be found'
+            );
+        }
+
+        if (!$raLocation instanceof RaLocation) {
+            throw new RuntimeException(
```
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Projector/RaLocationProjector.php]: Very thorough ;) Could you use the ApiBundle's RuntimeException, though? :)

**Comment by rjkip** [comment]
:+1:

File: src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Projector/RaLocationProjector.php (hunk 1 above)
Comment (relaxnow) [inline src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Projector/RaLocationProjector.php]: Sure, done.

**Pull request merged**